src/zenml/api.py

```diff
@@ -46,5 +52,10 @@ def show(ngrok_token: Optional[str] = None) -> None:
         logger.debug(f"Tunneling dashboard from {url} to {ngrok_url}.")
         url = ngrok_url
 
-    url = f"{url}:{port}" if port else url
+    if prefill_username:
+        url = (
+            f"{url}:{port}/login?username={DEFAULT_USERNAME}" if port else url
```

Comment:
If someone were to configure their local server with external authentication support, will this mess with it in any way @stefannica?

Reply:
A local server with external authentication support is not supported and not even technically possible, not even for development purposes.